### DISCUSSION TITLE
can we have something to categorize bookmarks on daily dev

### DISCUSSION THREAD

**Discussion by naren2605** [dailydotdev/daily#597] [Feature Requests]
can we have something to categorize bookmarks on daily dev

**Comment by Hoss3770** [comment]
Maybe you can allow users to create their own lists. Sth like youtube playlists. When saving the user can tick checkboxes to save to the lists of his choice.

**Comment by idoshamun** [comment]
Hi!

Let's keep the discussion here https://github.com/dailydotdev/daily/discussions/520.
Thanks!
